SysKit.XCellKit/SpreadsheetWorkbook.cs

```diff
@@ -1,28 +1,35 @@
-﻿using System;
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
-using DocumentFormat.OpenXml;
-using DocumentFormat.OpenXml.Packaging;
-using DocumentFormat.OpenXml.Spreadsheet;
 
 namespace SysKit.XCellKit
 {
     public class SpreadsheetWorkbook
     {
-        public SpreadsheetWorkbook() 
+        private string _tag = null;
+
+        public SpreadsheetWorkbook()
         {
             _worksheets = new List<SpreadsheetWorksheet>();
             _tableCount = 1;
         }
-        
-        private readonly List<SpreadsheetWorksheet> _worksheets; 
+
+        private readonly List<SpreadsheetWorksheet> _worksheets;
         public void AddWorksheet(SpreadsheetWorksheet spreadsheetWorksheet)
         {
             _worksheets.Add(spreadsheetWorksheet);
         }
 
+        public void AddTag(string tag)
```

Comment:
Maybe "SetTag". "Add" implies you can have multiple, not override the previous value.